docs/tutorials/on-device-training/ios-app.md

```diff
@@ -0,0 +1,987 @@
+                progressCallback(progress)
+            }
+        }
+        
+    }
+    ```
+
+3.  #### Exporting the trained model
+
+    We can use the `exportModelForInference` method of the `ORTTrainingSession` class to export the trained model. The method takes in the path where the model should be exported and the output names of the model.
+    
+    Here we will export the model to the `Library` directory of the application. The exported model will be used for inference purposes.
+
+    ```swift
+    func exportModelForInference() throws {
+        guard let libraryDirectory = FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask).first else {
+            throw TrainerError.Error("Failed to find library directory ")
+        }
+        
+        let modelPath = libraryDirectory.appendingPathComponent("inference_model.onnx").path
+        try trainingSession.exportModelForInference(withOutputPath: modelPath, graphOutputNames: ["output"])
+    }
+    ```
+
+You can find the complete implementation of the `Trainer` class [here](https://github.com/microsoft/onnxruntime-training-examples/blob/master/on_device_training/mobile/ios/MyVoice/Trainer.swift)
+
+
+### Inference with the trained model
+The `VoiceIdentifier` class will handle the inference with the trained model. It will load the trained model and perform inference on the given audio data. The class will have the `evaluate(inputData: Data) -> Result<(Bool, Float), Error>` method that will take in the audio data and return the result of the inference. The result will be a tuple of `(Bool, Float)`, where the first element represents whether the audio belongs to the user or not, and the second element represents the confidence score of the prediction.
+
+First, we load the trained model using `ORTSession` object. 
+
+```swift
+class VoiceIdentifier {
+    
+    private let ortEnv : ORTEnv
+    private let ortSession: ORTSession
+    private let kThresholdProbability: Float = 0.80
+    
+    enum VoiceIdentifierError: Error {
+        case Error(_ message: String)
+    }
+    
+    init() throws {
+        ortEnv = try ORTEnv(loggingLevel: ORTLoggingLevel.warning)
+
+        guard let libraryDirectory = FileManager.default.urls(for: .libraryDirectory, in: .userDomainMask).first else {
+            throw VoiceIdentifierError.Error("Failed to find library directory ")
+        }
+        let modelPath = libraryDirectory.appendingPathComponent("inference_model.onnx").path
+
+        if !FileManager.default.fileExists(atPath: modelPath) {
+            throw VoiceIdentifierError.Error("Failed to find inference model file.")
+        }
+        ortSession = try ORTSession(env: ortEnv, modelPath: modelPath, sessionOptions: nil)
+    }
+}
+```
+
+Next, we will write the `evaluate` method that will take in the audio data, convert it to `ORTValue`, and perform inference with the model. Then, take the output of the model and extract probabilities.
```

Comment:
```suggestion
Next, we will write the `evaluate` method that will take in the audio data, convert it to `ORTValue`, and perform inference with the model. Then, it will take the output of the model and extract probabilities.
```

Reply:
maybe explain a bit what kind of postprocessing is required for this model?